packages/lib-utils/src/app/redux/actions/k8s.ts

```diff
@@ -184,11 +182,10 @@ export const watchK8sList =
           return;
         }
 
-        const { subprotocols } = getImpersonate(getState()) || {};
         WS[id] = k8sWatch(
           k8skind,
           { ...queryWithCluster, resourceVersion },
-          { subProtocols: subprotocols, timeout: 60 * 1000 },
+          { timeout: 60 * 1000 },
```

Comment:
@vidyanambiar could we move this timeout value to a constant somewhere? Perhaps near the top of the file near `paginationLimit`? 

It might be good to also indicate paginationLimit is a constant by changing it to: `PAGINATION_LIMIT`

Reply:
@florkbr Thanks. I've pushed this change up.